test/Microsoft.AspNetCore.SignalR.Client.Tests/SyncPoint.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+// Possibly useful as a general-purpose async testing helper?
+public class SyncPoint
+        {
+            private TaskCompletionSource<object> _atSyncPoint = new TaskCompletionSource<object>();
```

Comment:
Formatting

Reply:
C#'s "Move type to file" refactoring derped it up :)